### PR TITLE
[CI:DOCS] Add more documentation on UID/GID Mappings with --userns=keep-id

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
Better documentation for --userns=keep-id:uid=X,gid=Y
```

### CODE REVIEW

File: docs/source/markdown/options/userns.container.md [hunk 1]
```diff
@@ -12,10 +12,11 @@ Rootless user --userns=Key mappings:
 
 Key       | Host User |  Container User
 ----------|---------------|---------------------
-""        |$UID           |0 (Default User account mapped to root user in container.)
-keep-id   |$UID           |$UID (Map user account to same UID within container.)
-auto      |$UID           | nil (Host User UID is not mapped into container.)
-nomap     |$UID           | nil (Host User UID is not mapped into container.)
+""        |$UID         |0 (Default User account mapped to root user in container.)
+keep-id   |$UID         |$UID (Map user account to same UID within container.)
+keep-id:uid=200,gid=210 |$UID| 200:210 (Map user account to specified uid, gid value within container.)
```
Comment (mheon) [inline docs/source/markdown/options/userns.container.md:17]: Maybe make more generic - `uid=$uid,gid=$uid` and `Map UID and GID of user running podman to $uid and $gid in container`

Comment (rhatdan) [reply docs/source/markdown/options/userns.container.md:17]: That is what --userns=keep-id does by default.
This is more the case of running a container with a builtin user and you want the builtin user to match your UID.

**Review by giuseppe — APPROVED** [review]
LGTM

File: docs/source/markdown/podman-run.1.md.in [hunk 2]
```diff
@@ -577,6 +577,8 @@ $ podman run -v data:/data2 -i -t fedora bash
 $ podman run -v /var/cache/dnf:/var/cache/dnf:O -ti fedora dnf -y update
 
 $ podman run -d -e MYSQL_ROOT_PASSWORD=root --user mysql --userns=keep-id -v ~/data:/var/lib/mysql:z,U mariadb
+
+$ podman run -d -e MYSQL_ROOT_PASSWORD=root --user mysql --userns=keep-id:uid=999,gid=999 -v ~/data:/var/lib/mysql:z mariadb
```
Comment (vrothberg) [inline docs/source/markdown/podman-run.1.md.in:581]: A comment above elaborating on the user and group would be helpful.

Comment (rhatdan) [reply docs/source/markdown/podman-run.1.md.in:590]: Done

**Review by vrothberg — COMMENTED** [review]
/lgtm

**Pull request merged**